### DISCUSSION TITLE
Avatars instead of moving heads?

### DISCUSSION THREAD

**Discussion by Therealjosephchrzempiec** [networked-aframe/networked-aframe#426] [General]
Hello, I just figure out how to add a 3d avatar in gltf format. I was wondering if I have multiple 3d avatars and have a button to change them would others see what change as well or do I have to refresh the page for them to see it?

Joseph

**Comment by vincentfretin** [comment]
You will need to handle setting the gltf-model similiar to how the username is set on the player-info via a component property and updating it in the update method.

**Reply by Therealjosephchrzempiec** [reply]
Thank you. I'm working on that.

**Comment by vincentfretin** [comment]
I wrote a complete example here https://github.com/networked-aframe/naf-valid-avatars

**Comment by Therealjosephchrzempiec** [comment]
Very nice man.
